src/config/mod.rs

```diff
@@ -36,6 +46,98 @@ impl OutputConfig {
     }
 }
 
+pub fn generate_config() -> PathBuf {
+    println!("Would you like to generate a config file? [y/N]");
+    let mut input = String::new();
+    io::stdin()
+        .read_line(&mut input)
+        .expect("Failed to read input");
+
+    if input.trim() == "y" {
+        println!("OK, generating config...");
+        let xdg = xdg::BaseDirectories::new().expect("Couldnt get xdg basedirs");
+        let file_path = xdg
+            .place_config_file("magamawm/config.ron")
+            .expect("Failed to get file path");
+        let mut file = match File::create(file_path.clone()) {
+            Ok(file) => file,
+            Err(err) => {
+                println!("Failed to create file: {}", err);
+                panic!("Couldnt create config file")
+            }
+        };
+
+        let mut keybinding_map = std::collections::HashMap::<KeyPattern, Action>::new();
+        keybinding_map.insert(
+            KeyPattern {
+                modifiers: KeyModifiersDef(vec![KeyModifier::Super]).into(),
+                key: xkb::KEY_Return,
+            },
+            Action::Spawn(String::from("alacritty")),
```

Comment:
kitty is more universally used by default by window managers. use that

Reply:
nah

Reply:
> kitty is more universally used by default by window managers. use that

Both ideas are stupid just use the default terminal / like i3 determine the installed one